src/casp/services/db/migrations/versions/t_2024-10-23_13-39-07_quota_increase.py

```diff
@@ -0,0 +1,25 @@
+"""quota_increase
+
+Revision ID: cdfe110d1068
+Revises: c80bd73203d1
+Create Date: 2024-10-23 13:39:07.609718
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "cdfe110d1068"
+down_revision = "c80bd73203d1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.add_column("users_user", sa.Column("quota_increased", sa.Boolean(), nullable=False, server_default="False"))
```

Comment:
I think that if you ran this migration in the db where users exist, it'd raise error, because the field is not nullable. I had this issue, but I didn't use server_default. Still, my understanding is that it is not going to apply default on fly to all existing entities. Please, correct me if I'm wrong if you tested this particular case. 
Otherwise, you can take a look at migration`37fbe8997eca_cas_model_is_default.py` there's a workaround solution for this.

Reply:
Also, I believe since we're using postgres it'd be better to put "false" as a server_default. Not sure 100% tho, but my understanding is that this is going directly into the database and I am not sure that postgres would accept 'False' (capitalized). This is what their docs say about their booleans:
https://www.postgresql.org/docs/6.3/c0805.htm

Reply:
I can confirm this works.  Using "server_default" is the solution I found to the error you described.  Also, postgres treats booleans as case-insensitive.  I think maybe all keywords are case-insensitive?

Reply:
Oh, that's a neat solution then! About case-insensitivity - cool, then I am resolving this message.

Reply:
This conversation got very esoteric all of a sudden. I see myself out.